src/Enzo/enzo_EnzoMethodMHDVlct.hpp

```diff
@@ -94,6 +81,20 @@ class EnzoMethodMHDVlct : public Method {
   /// @ingroup  Enzo
   /// @brief    [\ref Enzo] Encapsulate VL + CT MHD method
 
+public:
+  /// This is defined within the scope of EnzoMethodMHDVlct to avoid polluting
+  /// the global namespace.
+  ///
+  /// @note
+  /// This must be public so that it can be passed to helper methods by value
+  enum bfield_choice {
```

Comment:
Could probably be made an `enum class`

Reply:
As I recall, there are some weird issues with `PUP`ing instances of `enum class`. In particular I think I encountered an issue on Frontera or Stampede2 while using the intel compiler.